src/content/community/Community-RunningProjects.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Project resources
+layout: docs
+category: Community
+permalink: /community/project-resources/
+next: /community/upcoming-events/
+sublinks: Types of GraphQL projects,Running project meetings,Resources for projects
+---
+
+The following resources are intended to help official GraphQL projects understand what resources are available to them, and how to correctly run meetings..
+
+## Types of GraphQL projects
+
+The GraphQL community is incredibly broad, with a large number of reference implementations which are developed and evolving in parallel to the GraphQL project. Many of them are listed [here](/code/).
+
+The primary distinction between official GraphQL projects and other implementations is that GraphQL projects are managed under the [GraphQL technical charter](https://technical-charter.graphql.org) and [GraphQL Code of Conduct](https://code-of-conduct.graphql.org).
+
+We welcome and encourage community members and third-parties to maintain their own implementations.
+
+> ### Making an independent project official
+> 
+> If you maintain a tool or reference implementation that you'd like to transfer to the [GraphQL GitHub org](https://github.com/graphql), or if you are planning to start a new workgroup or implementation within the official GraphQL project, please add your name to the agenda of an upcoming [GraphQL Working Group meeting](https://github.com/graphql/graphql-wg/tree/HEAD/agendas) to introduce your idea.
+> 
+> A few things will then need to happen. For example, we will work with you to adopt our policies and neutral governance, and help you transfer the project and its assets (generally the domain name and any trademarks) so that they are neutrally held at the Linux Foundation. If you'd like to explore the idea before making a proposal, please reach out to [operations@graphql.org](mailto:operations@graphql.org)
```

Comment:
This makes it feel like you're definitely going to be accepted; I've caveated it:

```suggestion
> Should your proposal meet with approval, a few things will then need to happen. For example, we will work with you to adopt our policies and neutral governance, and help you transfer the project and its assets (generally the domain name and any trademarks) so that they are neutrally held at the Linux Foundation. If you'd like to explore the idea before making a proposal, please reach out to [operations@graphql.org](mailto:operations@graphql.org)
```

Reply:
Further, is it correct that the Linux Foundation holds the assets, rather than the GraphQL Foundation? I'm guessing it is correct due to the way the GF/LF relate, but wanted to make sure it wasn't a copy/paste error just in case.

Reply:
Yep, that's accurate. LF is the actual legal entity and thus the one that holds things like trademarks, whereas GraphQL Foundation is technically a fund within the LF.